src/org/thoughtcrime/securesms/ConversationItem.java

```diff
@@ -648,7 +648,7 @@ public void onClick(final View v, final Slide slide) {
 
         context.startActivity(intent);
       } else if (slide.getUri() != null) {
-        dcContext.openForViewOrShare(slide.getDcMsgId(), Intent.ACTION_VIEW);
+        ApplicationDcContext.openForViewOrShare(context, dcContext, slide.getDcMsgId(), Intent.ACTION_VIEW);
```

Comment:
I would use an static import here (less distracting text, and with a click I can see where it origins from), but that's only a matter of style.

Reply:
wait. Actually I agree with hugotty here. I think it would be cleaner to keep it an object function. With the newimplemented instance check, passing the Context will be obvious, too